src/presets/universal_deployer.cairo

```diff
@@ -0,0 +1,80 @@
+use starknet::ClassHash;
+use starknet::ContractAddress;
+
+#[starknet::interface]
+trait IUniversalDeployer<TState> {
```

Comment:
What do you think about moving this bottom? Or maybe we can create a UDC folder just with the interface. It looks odd for users to import the dispatcher from the preset file while the rest of the library uses the interface file.

Reply:
> It looks odd for users to import the dispatcher from the preset file while the rest of the library uses the interface file.

Yeah, that's a good point. Will update

Reply:
Done! Let me know what you think